e2e/e2e_suite_test.go

```diff
@@ -0,0 +1,152 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/medik8s/node-healthcheck-operator/api/v1alpha1"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/intstr"
+
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+func TestE2e(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "E2e Suite")
+}
+
+var dynamicClient dynamic.Interface
+var clientSet *kubernetes.Clientset
+
+var poisonPillTemplateGVR = schema.GroupVersionResource{
+	Group:    "poison-pill.medik8s.io",
+	Version:  "v1alpha1",
+	Resource: "poisonpillremediationtemplates",
+}
+var poisonPillGVR = schema.GroupVersionResource{
```

Comment:
maybe rename to `poisonPillRemdiationGVR`

Reply:
done